### PR TITLE
Prelint cleanup #397

### PR DESCRIPTION
**Pull request by legobeat**
Minor fixups to make the new linting configuration in #397 pass. Not much! :medal_sports:

### CODE REVIEW

**Review by naugtur — APPROVED** [review]

File: packages/viz/src/App.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ import * as mergePolicy from './merge-deep.js'
 import Nav from './views/nav.js'
 
 
-const { DepGraph } = require('./graphs/DepGraph')
+const { DepGraph } = require('./graphs/DepGraph.js')
```
Comment (naugtur) [inline packages/viz/src/App.js:8]: This line is weird... 
@kumavis why is there a mix of import and require in one file?

Comment (legobeat) [reply packages/viz/src/App.js:8]: Indeed. Do we want to consider enabling `no-var-requires`/`no-require-imports` as well?

**Comment by legobeat** [comment]
Updated to comply with new rules in #397

File: packages/viz/src/App.js (hunk 1 above)
Comment (kumavis) [reply packages/viz/src/App.js:8]: haha idk! viz is getting old and dusty

**Comment by kumavis** [comment]
merged in #409

**Pull request closed without merging**